hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -82,20 +82,27 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
 
         final var fileStore = handleContext.writableStore(WritableFileStore.class);
         final var fileUpdate = handleContext.body().fileUpdateOrThrow();
-        final var maybeFile = fileStore.get(fileUpdate.fileIDOrElse(FileID.DEFAULT));
 
         final var fileServiceConfig = handleContext.configuration().getConfigData(FilesConfig.class);
 
         if (fileUpdate.fileID() == null) {
             throw new HandleException(INVALID_FILE_ID);
         }
 
-        // the update file always will be for the node, not a particular ledger that's why we just compare th num
+        // the update file always will be for the node, not a particular ledger that's why we just compare the fileNum
+        // and ignore shard and realm
         if (fileUpdate.fileIDOrThrow().fileNum() == fileServiceConfig.upgradeFileNumber()) {
             handleUpdateUpgradeFile(fileUpdate, handleContext);
             return;
         }
 
+        // copy over just the fileNum from the update file
```

Comment:
This isn't an update file, at this point the code is working with a normal file.
Upgrade file returns on line 95.

Reply:
fixed